scout/quickstart.md

```diff
@@ -0,0 +1,210 @@
+and then the CVE ID (in this case, "CVE-2022-24999⁠") to see details of the
+vulnerability.
+
+![Screenshot showing highlighting a CVE in Docker Desktop](./images/scout-onboarding-dd-v1-cvve-focus.png)
+
+Docker Scout creates and maintains its vulnerability database by ingesting and
+collating vulnerability data from multiple sources continuously. These sources
+include many recognizable package repositories and trusted security trackers.
+You can find more details in the [Advisory Database sources](./advisory-db-sources.md) document.
+
+You can also use the Docker Scout CLI to see the same results.
+
+```shell
+docker scout cves <org-name>/scout-demo:v1
+```
+
+![Screenshot showing highlighting a CVE in the Docker Scout CLI](./images/scout-onboarding-cli-v1.png)
+
+> **Tip**
+>
+> Find out how filter results [in the reference documentation for the CLI command](/engine/reference/commandline/scout_cves).
+> {: .tip }
+
+## Fix application vulnerabilities
+
+The fix suggested by Docker Scout is to update
+the underlying vulnerable express version to 4.17.3 or later. Do this by
+changing the `package.json` file.
+
+```json
+…
+  "dependencies": {
+    "express": "4.17.3"
+    …
+  }
+```
+
+Rebuild the image, giving it a new version tag:
+
+```shell
+docker build -t <org-name>/scout-demo:v2 .
+```
+
+Push the image it to the same repository on Docker Hub using a new version tag:
+
+```shell
+docker push <org-name>/scout-demo:v2
+```
+
+Now, viewing the latest tag of the image in Docker Desktop, the Docker Scout
+Dashboard, or CLI, you can see that you have fixed the vulnerability.
+
+## Fix vulnerabilities in base images
+
+In addition to identifying application
+vulnerabilities, Docker Scout also helps you identify and fix issues with the
+base images your images use.
+
+In Docker Desktop and the Docker Scout Dashboard, you can see these suggestions
+by selecting the **Recommended fixes** button in the image layer view.
+
+Select the **Recommendations for base image…** option from the button. In the
+dialogue that appears, select the **Change base image** tab, the new version of
+the base image you want to use, and copy the suggestion into your `Dockerfile`.
+
+Rebuild the image, again with a new tag:
+
+```shell
+docker build -t <org-name>/scout-demo:v3 .
+```
+
+Push it to Docker Hub using the new version tag:
+
+```shell
+docker push <org-name>/scout-demo:v3
+```
+
+Select the new image tag in Docker Desktop or the Docker Scout Dashboard and you
+can see that the base image has been updated, removing many vulnerabilities.
+
+You can see the same using the CLI command:
+
+```shell
+docker scout cves <org-name>/scout-demo:v3
+```
+
+## Collaborate on vulnerabilities
+
+You can see and share the same vulnerability information about an image and
+the other images in your organization in the [Docker Scout Dashboard](./dashboard.md).
+Developers can see an overview of all their images from Docker Hub and Artifactory,
```

Comment:
Not just developers I guess, but security champions etc that are part of the org can also get this information from the dashboard.

```suggestion
Organization members can see an overview of all their images from Docker Hub and Artifactory,
```